google/cloud/forseti/services/scanner/dao.py

```diff
@@ -183,6 +184,18 @@ def __repr__(self):
         return string.format(
             self.violation_type, self.resource_type, self.rule_name)
 
+    @staticmethod
+    def get_schema_update_actions():
+        """Maintain all the schema changes for this table.
+
+        Returns:
+            dict: A mapping of Action: Column.
+        """
+        column_mapping = {'CREATE': [Column('resource_name',
```

Comment:
Can we move the list out, so that it's easier to read?

```
columns_to_create = [
    Column('resource_name',  String(256), default=''),
    Column('abc',  String(256), default=''),
    Column('xyz',  String(256), default=''),
]

column_mapping = {
    'CREATE': columns_to_create,
    DELETE: columns_to_delete,
}
```

Reply:
Updated